### DISCUSSION TITLE
counts by uri

### DISCUSSION THREAD

**Discussion by mlybarger** [martin-helmich/prometheus-nginxlog-exporter#210] [Q&A]
hi,  great work on this project!  

i'm interested in seeing metrics from my nginx server by token within the request uri.  the uri's might be like

/app/east/token/foo
/app/east/token/bar
/app/west/token/foo
/app/west/token/bar

so, i wan to see east status (req/sec?) and west stats (and north and south too).   any thoughts on how to do this easily?

**Comment by stempler** [comment]
Did you check the second example in the [Dynamic relabeling](https://github.com/martin-helmich/prometheus-nginxlog-exporter#dynamic-re-labeling) section of the README?
That goes into the direction you want I think.
